src/main/java/seedu/address/storage/JsonAdaptedAppointment.java

```diff
@@ -21,18 +24,22 @@
     private final String address;
     private final DateTime date;
     private final Set<JsonAdaptedPerson> contacts = new HashSet<>();
-    // private final List<JsonAdaptedTag> tagged = new ArrayList<>();
+    private final List<JsonAdaptedTag> tagged = new ArrayList<>();
```

Comment:
Maybe change name to tags instead, since it is a list?

Reply:
yup, or can we use taggedList?